vllm/entrypoints/openai/api_server.py

```diff
@@ -131,6 +131,9 @@ async def build_async_engine_client(args) -> AsyncIterator[AsyncEngineClient]:
         logger.info("Multiprocessing frontend to use %s for RPC Path.",
                     rpc_path)
 
+        # Build RPCClient, which conforms to AsyncEngineClient Protocol.
```

Comment:
moved first, since we `bind` here
- note: this shouldn't matter, but just being safe